src/components/implementation/no_interface/llbooter/llbooter.c

```diff
@@ -227,38 +229,39 @@ static void
 boot_init_sched(void)
 {
 	int i;
-	
-	for (i = 0 ; i < MAX_NUM_SPDS ; i++) schedule[i] = 0;
+
+	for (i = 0; i < MAX_NUM_SPDS; i++)
+		schedule[i] = 0;
 	sched_cur = 0;
 }
-						
+
 static void
 boot_create_cap_system(void)
 {
 	unsigned int i;
-	
-	for (i = 0 ; hs[i] != NULL ; i++) {
+
+	for (i = 0; hs[i] != NULL; i++) {
 		struct cobj_header *h;
 		struct cobj_sect *sect;
 		captblcap_t ct;
 		pgtblcap_t pt;
 		spdid_t spdid;
 		vaddr_t ci = 0;
-		
-		h = hs[i];
+
+		h     = hs[i];
 		spdid = h->id;
-		printc("Initializing Comp: %s\n", h->name);	
-		
-		sect = cobj_sect_get(h, 0);
+		printc("Initializing Comp: %s\n", h->name);
+
+		sect                                = cobj_sect_get(h, 0);
```

Comment:
I guess it does line up `=`s, but only `=`s (not `-=` and such).  It is likely OK we lose that small capability.